### PR TITLE
electrum: Add version 4.0.2

### PR DESCRIPTION
**Pull request by erogy**
https://github.com/spesmilo/electrum

### CODE REVIEW

File: bucket/electrum.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    ],
+    "shortcuts": [
+        [
+            "electrum.exe",
+            "Electrum"
+        ]
+    ],
+    "persist": "electrum_data",
+    "checkver": {
+        "github": "https://raw.githubusercontent.com/spesmilo/electrum-web/master/panel-download.html",
```
Comment (Ash258) [inline bucket/electrum.json:22]:
```suggestion
        "url": "https://raw.githubusercontent.com/spesmilo/electrum-web/master/panel-download.html",
```

File: bucket/electrum.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "description": "A Bitcoin SPV Wallet",
+    "homepage": "https://electrum.org",
+    "license": "MIT",
+    "url": "https://download.electrum.org/4.0.2/electrum-4.0.2-portable.exe#/electrum.exe",
+    "hash": "2e04699258027df8dd66335ae58567f81b45d01f5d2d2ab69da9ae037f39d1d2",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\")) {",
+        "   New-Item \"$dir\\electrum_data\\config\" -Force | Out-Null",
+        "   Set-Content \"$dir\\electrum_data\\config\" '{ \"check_updates\": false }' -Encoding Ascii",
+        "}"
```
Comment (Ash258) [inline bucket/electrum.json:12]:
```suggestion
        "if (!(Test-Path \"$persist_dir\\electrum_data\\config\")) {",
        "   ensure \"$dir\\electrum_data\" | Out-Null",
        "   Set-Content \"$dir\\electrum_data\\config\" '{ \"check_updates\": false }' -Encoding Ascii",
        "}"
```

**Comment by erogy** [comment]
> How does this differ from electron-cash?

electron-cash is a fork of electrum hence their similarity. Electrum is for Bitcoin, electron-cash is for Bitcoin Cash.

**Pull request merged**